project/vision_backend_api/tests/test_deploy.py

```diff
@@ -400,7 +402,8 @@ def test_pre_extract(self):
                 image_order=0),
             "Unit's request_json should be correct")
 
-    @patch('vision_backend_api.tasks.deploy_classify.run', noop_task)
+    @skip("Deprecated since deploy now a single call.")
```

Comment:
@StephenChan :  Did I understand this correctly that this is deprecated due to single call? If so, should I just delete it?

Reply:
I think `test_pre_extract` and `test_pre_classify` should be replaced with a `test_pre_deploy` test. It should be quite similar to `test_pre_extract`: check that the ApiJob was created with correct field values, and check that the ApiJobUnit was created with correct field values (the unit type should now be deploy instead of feature extract).

Reply:
Fixed.

Reply:
Looks good, thanks.